backend/uclapi/dashboard/tests.py

```diff
@@ -442,3 +446,112 @@ def test_refresh_verification_secret_POST_success(
         self.assertEqual(response.status_code, 200)
         self.assertTrue(content["ok"])
         self.assertTrue("new_secret" in content.keys())
+
+
+def post_request_only(self, url, view):
+    request = self.factory.get(
+        url,
+        {
+        }
+    )
+
+    response = view(request)
+    content = json.loads(response.content.decode())
+    self.assertEqual(response.status_code, 400)
+    self.assertEqual(
+        content["error"],
+        "Request is not of method POST"
+    )
+
+
+def empty_get_request_only(self, url, view, error):
+    request = self.factory.post(
+        url,
+        {
+        }
+    )
+
+    response = view(request)
+    content = json.loads(response.content.decode())
+    self.assertEqual(response.status_code, 400)
+    self.assertEqual(
+        content["message"],
+        error
+    )
+
+
+class api_applicationsTestCase(TestCase):
```

Comment:
```suggestion
class ApiApplicationsTestCase(TestCase):
```
Keeps it in line with our other class names :)